scripts/encode_data.py

```diff
@@ -93,7 +93,7 @@ def _load_files(self) -> None:
                 name, extension = os.path.splitext(full_path) 
 
                 match extension:
-                    case ".pdf" | ".rst":
+                    case ".pdf" | ".rst" | ".sh" | ".conf" | ".md":
```

Comment:
If you add these filetypes here you should add the loaders in the process functon. Maybe this part should be a seperate pr

Reply:
To explain why we need to do this is because it's not done automatically by filetype, its done explicitly in our process function